hirte.spec.in

```diff
@@ -12,6 +12,9 @@ BuildRequires: systemd-devel
 BuildRequires: systemd-rpm-macros
 BuildRequires: golang-github-cpuguy83-md2man
 
+BuildRequires:  python3-devel
+Buildrequires:  python3dist(dasbus)
+
```

Comment:
Please move those BuildRequires only under python3-hirte package, we may disable building python package from production builds